### ISSUE TITLE
Search exclude folders stopped working

### ISSUE THREAD

**Issue by ohadco** [microsoft/vscode#95757]
Issue Type: <b>Bug</b>

I started seeing excluded directories under search (node_modules, build)
folders which are excluded under settings --> features --> search --> Exclude.
I have `**/node_modules` and `**/build` excluded there, but started seeing them in the search results a few days ago

VS Code version: Code 1.44.2 (ff915844119ce9485abfe8aa9076ec76b5300ddd, 2020-04-16T17:07:18.473Z)
OS version: Darwin x64 17.2.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz (8 x 2800)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|16.00GB (0.51GB free)|
|Process Argv|-psn_0_7534383|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (22)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-javascript-snippet-pack|aka|0.1.5
path-intellisense|chr|1.4.2
vscode-eslint|dba|2.1.5
gitlens|eam|10.2.1
prettier-vscode|esb|4.4.0
relativepathrefactor|jak|0.9.0
sublime-babel-vscode|jos|0.2.10
dotenv|mik|1.0.1
prettify-json|moh|0.0.3
python|ms-|2020.3.71659
debugger-for-chrome|msj|4.12.6
vscode-react-native|msj|0.14.2
storybook-snippets|nor|0.0.5
vetur|oct|0.24.0
vscode-jest|Ort|3.1.1
ruby|reb|0.27.0
vscode-yaml|red|0.7.2
ActiveFileInStatusBar|Ros|1.0.3
python|tht|0.2.3
full-react-snippets|wal|1.4.1
vscode-ruby|win|0.27.0
JavaScriptSnippets|xab|1.7.2

</details>

**Comment by IllusionMH** [comment]
/needsMoreInfo

Are you sure that you didn't disable "Use Exclude Settings and Ignore Files" in files to exclude filed?
It should have background

![image](https://user-images.githubusercontent.com/1312662/79853913-52df7080-83d1-11ea-80ac-9eba40c057cc.png)

**Comment by fongoscar** [comment]
Version: 1.44.2
Commit: ff915844119ce9485abfe8aa9076ec76b5300ddd
OS: Darwin x64 18.7.0

I had similar but opposite issue.

I am able to use "files to exclude" to exclude files, but the "Use Exclude Settings and Ignore Files" button does not toggle off "files to exclude".

**Comment by roblourens** [comment]
Check that button and also see these tips: https://github.com/microsoft/vscode/wiki/Search-Issues

**Comment by fongoscar** [comment]
> Check that button and also see these tips: https://github.com/microsoft/vscode/wiki/Search-Issues

- No "files to exclude" -> files are not excluded.
- With "files to exclude" + "Use Exclude Settings and Ignore Files" ON -> files are excluded.
- With "files to exclude" + "Use Exclude Settings and Ignore Files" OFF -> files are excluded.

Apparently the "Use Exclude Settings and Ignore Files" is not properly functioning in my case.

**Comment by schroef** [comment]
Same issue here, tried so many possible ways. The folder "_Revisions" is added to the workspace. Ive added all these variations i could think of and none work, the are also added to ignore ile. Still they show up when in exclude is turned on
Weird thing is, i had this same issue before with a GitHub folder. After 3 tries i got that to work. Could the underscore be causing issues?
![Screen Shot 2020-08-01 at 4 28 02 PM](https://user-images.githubusercontent.com/6923008/89110001-3f05ca80-d414-11ea-81a2-43ff508a8fe5.png)

**Comment by schroef** [comment]
Tried renaming the folder to "00 Revisions" and "Revisions" add all combos to both workspace and user. Still keeps showing WTF!!! Reloaded multiple times, still keeps showing

![Screen Shot 2020-08-01 at 4 38 15 PM](https://user-images.githubusercontent.com/6923008/89110121-9b1d1e80-d415-11ea-88b5-d79bb89fb73f.png)
![Screen Shot 2020-08-01 at 4 38 27 PM](https://user-images.githubusercontent.com/6923008/89110122-9ce6e200-d415-11ea-9dac-07763d4084c6.png)





![Screen Shot 2020-08-01 at 4 38 15 PM](https://user-images.githubusercontent.com/6923008/89110115-8a6ca880-d415-11ea-9b55-cb3ec4835ccb.png)

I use added folders in the workspace. I got this other setup which does work. I think i missed something with adding proper naming perhaps??


![Screen Shot 2020-08-01 at 4 40 59 PM](https://user-images.githubusercontent.com/6923008/89110157-e800f500-d415-11ea-8007-ef2d8953e3fc.png)
